### DISCUSSION TITLE
API URL is not working after using docker-slim image

### DISCUSSION THREAD

**Discussion by rajeshjatavkonvergeai** [slimtoolkit/slim#260] [Q&A]
When running a python application with normal docker images, the application is running on Flask and the API URL is also working. When running this application with docker-slim image, the application is running but none of the APIs are working.

So please suggest what am I doing wrong creating docker image by docker-slim?

**Comment by kcq** [comment]
@rajeshjatavkonvergeai what kind of flask server app do you have? What's one of the endpoints? is there a database involved? What's the base path for your flask API? One of the options is to provide custom http probe commands to make calls to the API endpoints you have. The `--http-probe-cmd` and `--http-probe-cmd-file` flags can be used for that. Happy to help you setup those custom probe commands if you can share more information about your API.

**Reply by rajeshjatavkonvergeai** [reply]
@kcq Thanks for your response. I have a question. if I have used 10 APIs in my service then is it possible to reduce my image by docker-slim? If yes, then how to do this?

Please suggest with example.
